src/checkers/inference/InferenceLauncher.java

```diff
@@ -148,6 +148,7 @@ public void typecheck(String [] javaFiles) {
      */
     public void infer() {
         printStep("Inferring", outStream);
+        printStep("Sys Env: " + System.getenv(), outStream);
```

Comment:
How many lines does this output? Does it output all the environment variables even irrelevant ones?

Reply:
This is just a debug output. I try to see why I cannot pass the travis test.

Will remove this line once I figure out the reason.